### ISSUE TITLE
0.2.8版本bug

### ISSUE THREAD

**Issue by sakfire** [likelovewant/ollama-for-amd#10]
### What is the issue?

使用0.2.8版本运行qwen2模型，结果显示异常，退回0.2.7后可恢复正常。

### OS

Windows

### GPU

AMD

### CPU

AMD

### Ollama version

0.2.8

**Comment by likelovewant** [comment]
**Duplicate Question**: Please see [this issue](https://github.com/likelovewant/ollama/issues/7#issuecomment-2241370341) for a similar discussion.

**Upstream guidance**: For questions about Ollama, please direct your queries to the original upstream project
instead of this fork. This repository is primarily intended to add support for additional GPUs, rather than
introducing new features or modifying existing functionality related to Ollama.

**Comment by UncleUncleWang** [comment]
0.2.8版本及0.3.0的官方版本都可以支持qwen2，但是此分支版本的0.2.8与0.3.0都存在问题，现象跟0.2.6一致。回退到0.2.7之后确实是正常的，应该是本分支引入的问题。

**Comment by likelovewant** [comment]
> 0.2.8版本及0.3.0的官方版本都可以支持qwen2，但是此分支版本的0.2.8与0.3.0都存在问题，现象跟0.2.6一致。回退到0.2.7之后确实是正常的，应该是本分支引入的问题。

Test the issue with Ollama.

**Steps Taken:**

* I cloned the Ollama repository using `git clone`.
* I rebuilt Ollama from source.

**Problem:** Despite rebuilding, the issue persists. This problem originates from the original Ollama script
. The specific reason for this issue is unknown.

**Workaround:**  Interestingly, running `ollama.exe serve` directly (without packaging) allows successful model
downloading. However, when packaged, new models fail to download." should let ollama fix it .
